RecoHI/HiCentralityAlgos/plugins/CentralityTableProducer.cc

```diff
@@ -118,8 +118,8 @@ void CentralityTableProducer::endJob() {
     // Get values from root file
     CB = (CentralityBins*)inputTFile_->Get(Form("%s/run%d", rootTag_.data(), runnum_));
     cout << rootTag_.data() << endl;
-    CT = new CentralityTable();
-    CT->m_table.reserve(CB->getNbins());
+    CentralityTable CT;
```

Comment:
`CT` is still defined as a class member: L62 of this file should be removed

Reply:
@perrotta thanks for spotting this! I deleted it in 943101c